### DISCUSSION TITLE
New version - duplicate loras and no preview.

### DISCUSSION THREAD

**Discussion by mak-cz** [Comfy-Org/ComfyUI-Manager#1221] [Q&A]
Hello, I would like to ask if someone can advise me. We are using multiple ComfyUI clients running under the StabilityMatrix manager. Before the update I saw the screenshots of all LORA. After the update in the new manager I don't see any previews in LORA (and I have CivitApi filled in) and in the old manager I have all LORA twice once with previews and once without previews. And where did the option to download a new hotspot or lora go? Thank you 
<img width="1900" alt="2024-11-06_23h25_15" src="https://github.com/user-attachments/assets/cdbf68f4-ac72-41b5-a513-df3857026e58">

**Comment by ltdrdata** [comment]
The model manager is not a feature of ComfyUI-Manager.
You should inquire about this feature in the respective repository.
